docs/pbc/all/emails/202404.0/install-the-mailing-and-notifications-feature.md

```diff
@@ -102,3 +108,18 @@ class MailDependencyProvider extends SprykerMailDependencyProvider
 ```
 
 To verify that everything is set up correctly and send an email, see [How to create and register MailTypeBuilderPlugin](/docs/pbc/all/emails/{{page.version}}/howto-create-and-register-a-mail-type-builder-plugin.html).
+
+## Migration from SwiftMailer to Symfony Mailer
+
+If your application was using Swift Mailer before, and you also have SPRYKER_SMTP_ENCRYPTION='tls', please follow the next steps.
+
+1. Force the port for SMTP to 465 and release this change with the migration release:
+```php
+$config[SymfonyMailerConstants::SMTP_PORT] = '465';
+```   
+2. Create a support ticket to change **SPRYKER_SMTP_PORT** to 465
+
+3. Update the code and release it after the support ticket is resolved:
```

Comment:
mb just `Revert configuration to the previous state in order to use a port from the env variable`or smth like that?

Reply:
fixed